website/gatsby-config.js

```diff
@@ -0,0 +1,68 @@
+const path = require('path');
+
+module.exports = {
+  siteMetadata: {
+    siteUrl: 'https://rnfirebase.io',
+  },
+  plugins: [
+    `gatsby-plugin-typescript`,
+    `gatsby-plugin-react-helmet`,
+    `gatsby-plugin-emotion`,
+    {
+      resolve: `gatsby-plugin-sitemap`,
+      options: {
+        exclude: ['/about', 'about'],
+      },
+    },
+    {
+      resolve: `gatsby-plugin-google-tagmanager`,
+      options: {
+        id: `GTM-K2S593S`,
```

Comment:
Just notifying an identifier. I don't believe this is a problem though, unless there is some sort of abusive hit misattribution?

Reply:
Nah, it gets added as a script in the HTML anyway so no issues.